### ISSUE TITLE
high cpu usage?

### ISSUE THREAD

**Issue by vip3r011** [sethblack/javascript-fortuna#6]
actualy more of a question , why is usage so high 15% to 25% with advanced example with entropy?

**Comment by sethblack** [comment]
```
    const cpuSpeed = await si.cpu();
    const processes = await si.processes();
    const disksIO = await si.disksIO();
    const memory = await si.mem();
```

These are all expensive calls.

**Comment by vip3r011** [comment]
any good alternatives? and what time is maximum recomended for entropy?

**Comment by sethblack** [comment]
The purpose isn't to be particularly fast or efficient but to give you less predictable pseudo-random numbers. The new seed that is generated could just use one of the values or maybe another system-specific value, but it may be easier to guess the seed.

I only reseed the function when I'm generating long sets of random numbers, in my uses I feel safe generating a couple of hundred random values before a reseed is necessary but YMMV.

**Comment by vip3r011** [comment]
Thanx for info, cause I'm generating lot of random sequences , 1million plus .
Maybe a quantum Will work nicely ., for example [https://github.com/cbumgard/node-qrand](url)

**Comment by sethblack** [comment]
Yeah, fortuna will definitely eat up a lot of cpu while generating that many sequences.
